.github/workflows/builds.yml

```diff
@@ -276,21 +262,9 @@ jobs:
         run: >
           ccache -z &&
           cmake -B build -S .
-          -GNinja
-          -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-          -DCMAKE_BUILD_TYPE=Release
-          -DCMAKE_CXX_FLAGS=-Werror
+          --preset=github-ci
           -DCMAKE_CXX_STANDARD=${{ matrix.std }}
           -DCMAKE_INSTALL_PREFIX="${INSTALL_DIR}"
-          -DACTS_ENABLE_LOG_FAILURE_THRESHOLD=ON
-          -DACTS_BUILD_EVERYTHING=ON
-          -DACTS_BUILD_EXAMPLES_UNITTESTS=ON
-          -DACTS_BUILD_ODD=ON
-          -DACTS_BUILD_EXAMPLES_PYTHON_BINDINGS=ON
-          -DACTS_BUILD_EXAMPLES_EDM4HEP=ON
-          -DACTS_FORCE_ASSERTIONS=OFF
```

Comment:
I don't remember exactly why this was off anymore. In case we need it off here, it can still be overridden.